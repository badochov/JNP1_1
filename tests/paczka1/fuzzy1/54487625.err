Error in line 8: VjE S267 19,4
Error in line 5: A353 A956 83,7
Error in line 17: A353 S991 %9,7
Error in line 20: HTudBS S991 34,2
Error in line 22: VjE S991 59,9
Error in line 12: RL2v S267 18,5
Error in line 35: SNH8K S267 164,51
