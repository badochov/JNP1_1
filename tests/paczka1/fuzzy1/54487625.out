A353 S 31,6
RL2v A 76,2
VjE A 119,3
euE A 30,9 S 68,9
A353 226,4
S991 100,5
VjE A 119,3
A353 S 32,8
HTudBS A 0,8
RL2v A 160,0
VjE A 119,3
cDXzN A 91,5
euE A 30,9 S 68,9
A353 310,2
A956 92,3
S991 101,7
